### ISSUE TITLE
Undefined index in UselessParenthesesSniff.php

### ISSUE THREAD

**Issue by lptn** [slevomat/coding-standard#515]
```
 An error occurred during processing; checking has been aborted. The error message was: Undefined
|       |     index:  in
|       |     /path/vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/PHP/UselessParenthesesSniff.php
|       |     on line 258 (Internal.Exception)
```

This error also breaks other sniffs. As result -- false positive errors:
![image](https://user-images.githubusercontent.com/5278175/46475361-fe27d500-c7ed-11e8-9c1f-5a014ac6b8c6.png)

Code that leads to this error (simplified)
```php
return true
    ? 100
    : (int) ((100 / 50) * 100);
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/057f3f154cf4888b60eb4cdffadc509a3ae9dccd
